contrib/linearize/linearize-data.py

```diff
@@ -209,14 +210,14 @@ def run(self):
 				return
 			inLenLE = inhdr[4:]
 			su = struct.unpack("<I", inLenLE)
-			inLen = su[0] - 80 # length without header
+			self.inLen = su[0] - 80 # length without header
```

Comment:
I'm not sure making `inLen` an instance variable is the right solution to the problem.
Only one function outside `run()`, `writeBlock()`, uses `self.inLen`, which is never defined, but the on-disk size can be easily computed in the function itself with:
```
def writeBlock(self, inhdr, blk_hdr, rawblock):
    blockSizeOnDisk = len(inhdr) + len(blk_hdr) + len(rawblock)
    if not self.fileOutput and ((self.outsz + blockSizeOnDisk) > self.maxOutSz):
```